webapp/lib/controller.dart

```diff
@@ -438,15 +470,15 @@ void command(UIAction action, Data data) {
     case UIAction.userSignedOut:
```

Comment:
Should "universal" actions such as sign out be moved into the `command` function above?

Reply:
That would make sense, yes. I'll address this in a separate PR, thanks!